### PR TITLE
changed __tests__/praxis/arthur-deviation-biases/unit.ts to ts

### PR DESCRIPTION
**Pull request by ariain**
A GIF or MEME to give some spice of the internet

## Motivation and Context

This PR is for [issue](https://github.com/BrainJS/brain.js/issues/551) #551 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Author's Checklist:

- [ ] My code focuses on the main motivation and avoids scope creep.
- [ ] My code passes current tests and adds new tests where possible.
- [x] My code is [SOLID](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)) and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
- [ ] I have updated the documentation as needed.

## Reviewer's Checklist:
- [x] I kept my comments to the author positive, specific, and productive.
- [x] I tested the code and didn't find any new problems.
- [x] I think the motivation is good for the project.
- [x] I think the code works to satisfies the motivation.

### CODE REVIEW

**Comment by ariain** [comment]
@mubaidr please review this too and merge...

File: __tests__/praxis/arthur-deviation-biases/unit.ts [hunk 1]
```diff
@@ -34,14 +33,16 @@ describe('ArthurDeviationBiases Class: Unit', () => {
           momentum: 0.2,
         },
       });
-      const result = kernel(weights, deltas);
-      expect(shave(result)).toEqual(shave([[1.25, 2.20000005, 3.1500001]]));
+      const result: any = kernel(weights, deltas);
```
Comment (mubaidr) [inline __tests__/praxis/arthur-deviation-biases/unit.ts]: Use type for `result`

File: __tests__/praxis/arthur-deviation-biases/unit.ts [hunk 2]
```diff
@@ -51,24 +52,24 @@ describe('ArthurDeviationBiases Class: Unit', () => {
...
       const mockDeltas = 2;
-      const mockLayer = {
+      const mockLayer: any = {
         width: 2,
         height: 2,
         deltas: mockDeltas,
         weights: mockWeights,
       };
-      const p = new ArthurDeviationBiases(mockLayer);
+      const p: any = new ArthurDeviationBiases(mockLayer);
```
Comment (mubaidr) [inline __tests__/praxis/arthur-deviation-biases/unit.ts:61]: Use type for `p`

**Review by mubaidr — CHANGES_REQUESTED** [review]

File: __tests__/praxis/arthur-deviation-biases/unit.ts (hunk 2 above)
Comment (ariain) [reply __tests__/praxis/arthur-deviation-biases/unit.ts:61]: `Type 'Mock<{}, []>' is missing the following properties from type 'IKernelRunShortcut': kernel, exec, source, Kernel, and 67 more.`
This error is due to line 63.

File: __tests__/praxis/arthur-deviation-biases/unit.ts (hunk 1 above)
Comment (ariain) [reply __tests__/praxis/arthur-deviation-biases/unit.ts:36]: same problem here `Argument of type 'KernelOutput' is not assignable to parameter of type 'Float32Array'.
  Type 'number' is not assignable to type 'Float32Array'.` on line 38.

File: __tests__/praxis/arthur-deviation-biases/unit.ts (hunk 2 above)
Comment (ariain) [reply __tests__/praxis/arthur-deviation-biases/unit.ts:61]: `Type 'Mock<{}, []>' is missing the following properties from type 'IKernelRunShortcut': kernel, exec, source, Kernel, and 67 more.`
This error is due to line 63.

File: __tests__/praxis/arthur-deviation-biases/unit.ts (hunk 1 above)
Comment (ariain) [reply __tests__/praxis/arthur-deviation-biases/unit.ts]: `same problem here Argument of type 'KernelOutput' is not assignable to parameter of type 'Float32Array'. Type 'number' is not assignable to type 'Float32Array'`. on line 38.

File: __tests__/praxis/arthur-deviation-biases/unit.ts (hunk 2 above)
Comment (ariain) [reply __tests__/praxis/arthur-deviation-biases/unit.ts:61]: @mubaidr any update...

File: __tests__/praxis/arthur-deviation-biases/unit.ts (hunk 1 above)
Comment (mubaidr) [reply __tests__/praxis/arthur-deviation-biases/unit.ts]: You can cast it to `Float32Array` for now. 

This requires using generics in the base neural network class, which will be covered in another pull request.

File: __tests__/praxis/arthur-deviation-biases/unit.ts (hunk 2 above)
Comment (mubaidr) [reply __tests__/praxis/arthur-deviation-biases/unit.ts:61]: Skip this for now.

File: __tests__/praxis/arthur-deviation-biases/unit.ts (hunk 1 above)
Comment (ariain) [reply __tests__/praxis/arthur-deviation-biases/unit.ts]: done...

**Pull request merged**